report-builder/resources/htmlTemplates/macros.ftl

```diff
@@ -93,6 +93,7 @@
 <!DOCTYPE html>
 <html id="htmlId">
 <head>
+  <#if charset?has_content><meta http-equiv="Content-Type" content="text/html;charset=${charset}"> </#if>
```

Comment:
Do we actually need to set the charset different from `utf-8`? Won't it be enough just to add `<meta http-equiv="Content-Type" content="text/html;charset=utf-8">` line?

Reply:
In my opinion, most environments(with my env, too) will be covered by `UTF-8`. but some other environments should be need to use other charset. Like this [SOF question](https://stackoverflow.com/questions/1200063/why-does-anyone-use-an-encoding-other-than-utf-8), We can't be sure which environment users will use :)

Reply:
I also think explicitly enabling configuration is better than implicit 😄

Reply:
Currently, we generate an HTML document with the default charset. It is utf-8, I think. So, what is expected to happen if a user sets a different charset? I think, we have two options here:
1) Set utf-8 charset as the default with no option to change it
2) Add an option to change, but this also should affect the generated content: we must make sure that all of them use the same charset

What do you think?

Reply:
> Currently, we generate an HTML document with the default charset. It is utf-8, I think. 

- Default charset may differ depending on the browser. If the browser reads report as html4, default charset will be `ISO-8859-1`. with my env, it was read as html4 so some characters were broken. below is some example

<img width="248" alt="image" src="https://user-images.githubusercontent.com/34061042/229496896-c35e9f31-762d-4a6a-b931-97a43df40152.png">

> 2. Add an option to change, but this also should affect the generated content: we must make sure that all of them use the same charset

- I think that some user set the charset(not utf-8), it means because they know that they are using other charset to the projects, so they expect that all generated report would use the same charset. By upper reason, i don't concern about some side effects. Nevertheless, if you are concerned, I think that allowing set charset on a specific source file can be a good solution 👍